src/current/v23.2/configure-logs.md

```diff
@@ -257,9 +257,10 @@ Along with the [common sink parameters](#common-sink-parameters), each HTTP serv
 | `disable-keep-alives` | When `true`, disallows reuse of the server connection across requests.<br><br>**Default:** `false` (reuses connections)                                                                                                                                                                                                                                            |
 | `compression`         | Compression method for the HTTP request body. Valid values `gzip` or `none`.<br><br>**Default:** `gzip` |
 | `headers`             | Map of key-value string pairs which will be appended to every request as custom HTTP headers. |
+| `file-based-headers`  | Map of key-filepath string pairs. The key and the value contained in the filepath will be appended to every request as a custom HTTP header. Values found at filepaths can be updated without restarting the `cockroach` process. Send SIGHUP to the `cockroach` process to notify it to refresh values.<br><br>For example:<br>`file-based-headers: {X-CRDB-FILE-HEADER: /some/path/to/file, X-ANOTHER-FILE-HEADER: /other/path/to/file}`|
```

Comment:
> The key and the value contained in the filepath

nit: for a moment when I read this, I thought it implied that there would be a key *and* value in the file. Maybe it's just me, but can this be reworded slightly to make it clearer that the key in the YAML config, and only the value is in the file?

Reply:
Thanks for catching this ambiguity. I reworded to clarify.